ztp/gitops-subscriptions/argocd/example/siteconfig/example-sno.yaml

```diff
@@ -66,6 +66,8 @@ spec:
         bootMode: "UEFI"
         rootDeviceHints:
           wwn: "0x11111000000asd123"
+#       disk partition at `/var/lib/containers` with ignitionConfigOverride. See DiskPartitionContainer.md for more details
+#       ignitionConfigOverride: '{"ignition":{"version":"3.2.0"},"storage":{"disks":[{"device":"/dev/disk/by-id/wwn-0x6b07b250ebb9d0002a33509f24af1f62","partitions":[{"label":"var-lib-containers","sizeMiB":0,"startMiB":250000}],"wipeTable":false}],"filesystems":[{"device":"/dev/disk/by-partlabel/var-lib-containers","format":"xfs","mountOptions":["defaults","prjquota"],"path":"/var/lib/containers","wipeFilesystem":true}]},"systemd":{"units":[{"contents":"# Generated by Butane\n[Unit]\nRequires=systemd-fsck@dev-disk-by\\x2dpartlabel-var\\x2dlib\\x2dcontainers.service\nAfter=systemd-fsck@dev-disk-by\\x2dpartlabel-var\\x2dlib\\x2dcontainers.service\n\n[Mount]\nWhere=/var/lib/containers\nWhat=/dev/disk/by-partlabel/var-lib-containers\nType=xfs\nOptions=defaults,prjquota\n\n[Install]\nRequiredBy=local-fs.target","enabled":true,"name":"var-lib-containers.mount"}]}}'
```

Comment:
Is this an optional configuration (thus commented out is OK) or one which we now require in which case it should not be commented out.

Reply:
This is required for any SNO that wants to use IBU, so mandatory

Reply:
Made it mandatory (uncommented)